### PR TITLE
rake task to destroy duplicated badges

### PR DESCRIPTION
**Pull request by ZePedroResende**

### CODE REVIEW

**Comment by marcelo140** [comment]
What if it deleted all repeated badges instead of having to specify one? It's true that it's a n unlikely situation, but this way we wouldn't have to specify the badge name x) @0urobor0s

File: lib/tasks/destroy_duplicated_badge.rake [hunk 1]
```diff
@@ -0,0 +1,14 @@
+namespace :sei do
+  desc "Removes duplicated badge from user"
+
+  task :destroy_duplicated_badge, [:badge, :username] => :environment do |t, args|
+    badge_id  = Badge.find_by(codename: args[:badge]).id
+    user_id   = User.find_by(username: args[:username]).id
+    badges = BadgeAcquisition.where(badge_id: badge_id, user_id: user_id)
+    if badges.length >1
```
Comment (marcelo140) [inline lib/tasks/destroy_duplicated_badge.rake]: Space missing after `>`

File: lib/tasks/destroy_duplicated_badge.rake [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+      badges[1..-1].each do |badge|
```
Comment (marcelo140) [inline lib/tasks/destroy_duplicated_badge.rake]: badges.drop(1) ?

**Review by marcelo140 — CHANGES_REQUESTED** [review]

File: lib/tasks/destroy_duplicated_badge.rake (hunk 1 above)
Comment (marcelo140) [reply lib/tasks/destroy_duplicated_badge.rake]: And extra space before `=` in line 6

File: lib/tasks/destroy_duplicated_badge.rake (hunk 2 above)
Comment (ZePedroResende) [reply lib/tasks/destroy_duplicated_badge.rake]: This way we are certain that every last duplicated badge is deleted just to be safe

**Comment by ZePedroResende** [comment]
@marcelo140 thats a good point , on it

**Comment by ZePedroResende** [comment]
@marcelo140 ping

**Review by marcelo140 — APPROVED** [review]

**Pull request merged**